src/Asana/Dispatcher/OAuthDispatcher.php

```diff
@@ -2,7 +2,7 @@
 
 namespace Asana\Dispatcher;
 
-use \OAuth2;
+use OAuth2;
```

Comment:
The manual claims they are unnecessary. I was not part of any previous discussion where it was determined they were. It seems to work without it.